### PR TITLE
[DOC] Update docs for 24.02.0 release [skip ci]

### PR DESCRIPTION
**Pull request by SurajAralihalli**
This PR updates download docs for 24.02.0 release.

Signed-off-by: Suraj Aralihalli [<suraj.ara16@gmail.com>](mailto:suraj.ara16@gmail.com)

### CODE REVIEW

File: docs/download.md [hunk 1]
```diff
@@ -81,29 +82,27 @@ CUDA 11.8 through CUDA 12.0.
...
-* Introduced support for chunked reading of ORC files.
-* Enhanced support for additional time zones and added stack function support.
-* Enhanced performance for join and aggregation operations.
-* Kernel optimizations have been implemented to improve Parquet read performance.
-* RAPIDS Accelerator also built and tested with Scala 2.13.
-* Last version to support Pascal-based Nvidia GPUs; discontinued in the next release.
-* Introduced support for parquet Legacy rebase mode (spark.sql.parquet.datetimeRebaseModeInRead=LEGACY and spark.sql.parquet.int96RebaseModeInRead=LEGACY)
-* Introduced support for Percentile function.
-* Delta lake 2.3 support.
+* Discontinued support for Nvidia GPUs based on Pascal architecture.
```
Comment (SurajAralihalli) [inline docs/download.md:95]: @viadea @nvliyuan

Please provide suggestions for 24.02 release notes

Comment (viadea) [reply docs/download.md:95]: Added more reviewers to comment on what we should highlight in release notes for 24.02.

File: docs/download.md [hunk 2]
```diff
@@ -65,14 +68,24 @@ for your hardware's minimum driver version.
...
+	- [RAPIDS Accelerator for Apache Spark 24.02.0 - Scala 2.12 jar.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0.jar.asc)
 
-- **Scala 2.13:**
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.13 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/23.12.2/rapids-4-spark_2.13-23.12.2.jar)
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.13 jar.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/23.12.2/rapids-4-spark_2.13-23.12.2.jar.asc)
+	- **Scala 2.13:**
+	- [RAPIDS Accelerator for Apache Spark 24.02.0 - Scala 2.13 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/24.02.0/rapids-4-spark_2.13-24.02.0.jar)
+	- [RAPIDS Accelerator for Apache Spark 24.02.0 - Scala 2.13 jar.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/24.02.0/rapids-4-spark_2.13-24.02.0.jar.asc)
+
+- **arm64**
```
Comment (SurajAralihalli) [inline docs/download.md:81]: @jlowe what do you think of adding arm64 jars this way?

File: docs/download.md [hunk 3]
```diff
@@ -65,14 +68,24 @@ for your hardware's minimum driver version.
 ### RAPIDS Accelerator's Support Policy for Apache Spark
 The RAPIDS Accelerator maintains support for Apache Spark versions available for download from [Apache Spark](https://spark.apache.org/downloads.html)
 
-### Download RAPIDS Accelerator for Apache Spark v23.12.2
-- **Scala 2.12:**
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.12 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.12.2/rapids-4-spark_2.12-23.12.2.jar)
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.12 jar.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.12.2/rapids-4-spark_2.12-23.12.2.jar.asc)
+### Download RAPIDS Accelerator for Apache Spark v24.02.0
+- **x86**
```
Comment (jlowe) [inline docs/download.md:72]:
```suggestion
- **x86_64**
```

File: docs/download.md (hunk 2 above)
Comment (jlowe) [reply docs/download.md:81]: This layout is fine, could also do a table layout with processor, Scala version, jar link, and signature link as columns.

File: docs/download.md [hunk 4]
```diff
@@ -81,29 +94,27 @@ CUDA 11.8 through CUDA 12.0.
...
-* RAPIDS Accelerator also built and tested with Scala 2.13.
-* Last version to support Pascal-based Nvidia GPUs; discontinued in the next release.
-* Introduced support for parquet Legacy rebase mode (spark.sql.parquet.datetimeRebaseModeInRead=LEGACY and spark.sql.parquet.int96RebaseModeInRead=LEGACY)
-* Introduced support for Percentile function.
-* Delta lake 2.3 support.
+* Discontinued support for Nvidia GPUs based on Pascal architecture.
+* Set get_json_object functionality to disabled by default.
+* Implemented string comparison in AST expressions.
+* Expanded timezone support to include options beyond UTC.
+* Enhanced security by adding checksum for cached files in Filecache.
```
Comment (jlowe) [inline docs/download.md:111]: Checksums are disabled by default, and it's not really a security feature.  If an attacker can overwrite the cached data they can also overwrite the checksum.
```suggestion
* Optional checksums for cached files in the file cache.
```

File: docs/download.md [hunk 5]
```diff
@@ -81,29 +94,27 @@ CUDA 11.8 through CUDA 12.0.
...
+* Introduced support for Databricks 13.3 ML LTS.
+* Added support for parse_url functionality.
 * Qualification and Profiling tool:
```
Comment (jlowe) [inline docs/download.md:114]: The qualification and profiling tools are released separately and on an independent schedule.  Does it make sense to list things here vs. point to the tools download page and let it describe what the recent releases are and what they have?

**Comment by NvTimLiu** [comment]
Looks good to me, I'll take care of other Docs update, e.g. https://github.com/NVIDIA/spark-rapids/pull/10389

File: docs/download.md (hunk 5 above)
Comment (viadea) [reply docs/download.md:114]: How about referring to https://github.com/NVIDIA/spark-rapids-tools/releases @mattahrens @amahussein  ?

File: docs/download.md [hunk 6]
```diff
@@ -81,29 +94,27 @@ CUDA 11.8 through CUDA 12.0.
...
-* Introduced support for parquet Legacy rebase mode (spark.sql.parquet.datetimeRebaseModeInRead=LEGACY and spark.sql.parquet.int96RebaseModeInRead=LEGACY)
-* Introduced support for Percentile function.
-* Delta lake 2.3 support.
+* Discontinued support for Nvidia GPUs based on Pascal architecture.
+* Set get_json_object functionality to disabled by default.
+* Implemented string comparison in AST expressions.
+* Expanded timezone support to include options beyond UTC.
+* Enhanced security by adding checksum for cached files in Filecache.
+* Introduced support for Databricks 13.3 ML LTS.
+* Added support for parse_url functionality.
```
Comment (viadea) [inline docs/download.md:113]: parse_url support for PATH, QUERY, PROTOCOL.

**Review by viadea — CHANGES_REQUESTED** [review]

File: docs/download.md (hunk 6 above)
Comment (viadea) [reply docs/download.md:113]: Support for Lazy Quantifier for regular expression functions

Comment (viadea) [reply docs/download.md:113]: Support for format_number function

Comment (viadea) [reply docs/download.md:113]: Batching support for row-based bounded window functions

File: docs/download.md [hunk 7]
```diff
@@ -65,14 +68,13 @@ for your hardware's minimum driver version.
...
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.13 jar.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/23.12.2/rapids-4-spark_2.13-23.12.2.jar.asc)
+### Download RAPIDS Accelerator for Apache Spark v24.02.0
+| Processor | Scala Version | Download Jar | Download Signature |
+|-----------|---------------|--------------|--------------------|
+| x86_64    | Scala 2.12    | [RAPIDS Accelerator 2.12 v24.02.0](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0.jar) | [Signature](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0.jar.asc) |
+| x86_64    | Scala 2.13    | [RAPIDS Accelerator 2.13 v24.02.0](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/24.02.0/rapids-4-spark_2.13-24.02.0.jar) | [Signature](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/24.02.0/rapids-4-spark_2.13-24.02.0.jar.asc) |
+| arm64     | Scala 2.12    | [RAPIDS Accelerator 2.12 v24.02.0](#) | [Signature](#) |
+| arm64     | Scala 2.13    | [RAPIDS Accelerator 2.13 v24.02.0](#) | [Signature](#) |
```
Comment (jlowe) [inline docs/download.md:77]: These links need to be fixed

Comment (SurajAralihalli) [reply docs/download.md:77]: Yes, I'll keep that in mind. I don't know what these links are going to be as it's the first release. I'll update them in a different PR when we update gh-pages.

Comment (jlowe) [reply docs/download.md:77]: Check with @NvTimLiu, these should be flled out here, not hacked into gh-pages afterwards unless we're for some reason not releasing ARM binaries simultaneously?

Comment (SurajAralihalli) [reply docs/download.md:77]: Sure!

**Review by viadea — APPROVED** [review]

**Comment by viadea** [comment]
build

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

File: docs/download.md (hunk 7 above)
Comment (sameerz) [reply docs/download.md]: I think the arm jars will be include the cuda classifier, and will be named 

rapids-4-spark_2.12-24.04.0-cuda11-arm64.jar
rapids-4-spark_2.12-24.04.0-cuda12-arm64.jar

rapids-4-spark_2.13-24.04.0-cuda11-arm64.jar
rapids-4-spark_2.13-24.04.0-cuda12-arm64.jar

Related issue: https://github.com/NVIDIA/spark-rapids/issues/6881

File: docs/download.md [hunk 8]
```diff
@@ -65,14 +68,13 @@ for your hardware's minimum driver version.
...
+### Download RAPIDS Accelerator for Apache Spark v24.02.0
+| Processor | Scala Version | Download Jar | Download Signature |
+|-----------|---------------|--------------|--------------------|
+| x86_64    | Scala 2.12    | [RAPIDS Accelerator 2.12 v24.02.0](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0.jar) | [Signature](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0.jar.asc) |
+| x86_64    | Scala 2.13    | [RAPIDS Accelerator 2.13 v24.02.0](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/24.02.0/rapids-4-spark_2.13-24.02.0.jar) | [Signature](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/24.02.0/rapids-4-spark_2.13-24.02.0.jar.asc) |
+| arm64     | Scala 2.12    | [RAPIDS Accelerator 2.12 v24.02.0](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0-arm64.jar) | [Signature](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0-arm64.jar.asc) |
```
Comment (NvTimLiu) [inline docs/download.md:76]: @SurajAralihalli  As Sameer mentioned : https://github.com/NVIDIA/spark-rapids/pull/10387#discussion_r1484988693

The arm jar names are combined with CUDA classifiers,  

As this PR got merged, can you help follow it up? Thanks!

Comment (SurajAralihalli) [reply docs/download.md:76]: Drafted [10414](https://github.com/NVIDIA/spark-rapids/pull/10414) to fix this, thanks!

File: docs/download.md (hunk 7 above)
Comment (SurajAralihalli) [reply docs/download.md]: Drafted [10414](https://github.com/NVIDIA/spark-rapids/pull/10414) to fix this, thanks!

**Pull request merged**